test/rekt/resources/configmap/configmap.go

```diff
@@ -0,0 +1,71 @@
+/*
+Copyright 2021 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
Regarding your comment on slack: You need to intend this line by one tab and not by 4 spaces :shrug:

Reply:
:+1: